__tests__/cmds/versions.test.js

```diff
@@ -137,11 +135,32 @@ describe('rdme versions*', () => {
 
       const mockRequest = getApiNock().post('/api/v1/version').basicAuth({ user: key }).reply(400, errorResponse);
 
-      await expect(createVersion.run({ key, version, fork: '0.0.5' })).rejects.toStrictEqual(
-        new APIError(errorResponse)
-      );
+      await expect(createVersion.run(args)).rejects.toThrow(new APIError(errorResponse));
       mockRequest.done();
     });
+
+    describe('ci environments', () => {
+      beforeEach(() => {
+        process.env.CIRCLECI = true;
+      });
+
+      afterEach(() => {
+        delete process.env.CIRCLECI;
+      });
+
+      it('should prompt for missing required arguments if in a CI environment', async () => {
+        const mockRequest = getApiNock()
+          .get('/api/v1/version')
+          .basicAuth({ user: key })
+          .reply(200, [{ version }, { version }]);
+
+        await expect(createVersion.run({ key, version })).rejects.toThrow(
+          /We've detected you're running within a (.*) environment, please supply the following required arguments for your command instead of relying on our terminal prompt system: fork/
```

Comment:
Any reason why we can't just test for an exact match here?

Reply:
Because even though I'm setting `CIRCLECI` to the environment, `GITHUB_ACTIONS` is also present and `ci-info` picks that up first so this message says "We've detected you're running within a Github Actions environment" in CI and "within a CircleCI environment" locally.